platform/mellanox/mlnx-platform-api/sonic_platform/component.py

```diff
@@ -23,6 +24,26 @@
 CPLD_VERSION_FILE_PATTERN = '/var/run/hw-management/system/cpld[0-9]_version'
 CPLD_VERSION_MAX_LENGTH = 4
```

Comment:
@stephenxs IMHO, will be better to have as part of CPLD class

Reply:
fixed.